buildout.cfg

```diff
@@ -16,6 +16,7 @@ show-picked-versions = true
 setuptools = 33.1.1
 zc.buildout = 2.9.5
 stdlib-list = 0.4.0
+cached-property = 1.3.1
```

Comment:
In most cases, a regular `@property` would have been enough, btw. 

Such a `@cached_property` is mostly useful when something is computationally intensive (and only on the condition that data doesn't change). So it isn't strictly needed as a dependency.

(I *do* like it, of course).

Reply:
Given how to code is right now, without the caching, there would be several properties (especially in ``PackageMetadata``) that would be called over and over, so I think is a quite neat solution 👍